sdk/identity/azure-identity/TROUBLESHOOTING.md

```diff
@@ -283,6 +284,8 @@ Get-AzAccessToken -ResourceUrl "https://management.core.windows.net"
 |---|---|---|
 |`CredentialUnavailableException` raised with message. "IntelliJ Authentication not available. Please log in with Azure Tools for IntelliJ plugin in the IDE."| The Credential was not able to locate the cached token to use for authentication. | Ensure that you login on the  Azure Tools for IntelliJ plugin, that will populate the cache for the credential to pick up.
 
+## Troubleshoot `AzurePipelinesCredential` authentication issues
+
```

Comment:
is this meant to be empty ?

Reply:
Yes, waiting on copy from @KarishmaGhiya.